File: labaTRPOtransport/Implement/Person/Worker.java
package com.company.Person;

public class Worker extends Person {
    private int salary;

    public Worker(){}

    public Worker(String name, String surname, int age, int salary){
        super(name, surname, age);
        this.salary = salary;
    }

    public int getSalary() {
        return salary;
    }

    public void setSalary(int salary) {
        this.salary = salary;
    }
}

File: lab3/FIrstUMLLab/Worker.cs
namespace FIrstUMLLab
{
    public class Worker : Person
    {
        public int Salary { get; private set; }

        public Worker(int salary, string name, string surname, int age)
            : base(name, surname, age)
        {
            this.Salary = salary;
        }
    }
}
